doc/man7/provider-digest.pod

```diff
@@ -69,7 +69,7 @@ For example, the "function" OP_digest_newctx() has these:
      OSSL_get_OP_digest_newctx(const OSSL_DISPATCH *opf);
 
 B<OSSL_DISPATCH> arrays are indexed by numbers that are provided as
-macros in L<openssl-core_numbers.h(7)>, as follows:
+macros in C<openssl/core_numbers.h>, as follows:
```

Comment:
No

Reply:
I disagree, as I've written above.  I'm not going to continue to disagree :)  But do you really think someone will write a podpage like the one mentioned above?  You're more optimistic than I am.

Reply:
I have that intention, yes.

Reply:
I disagree but I restored the old link.